docs/sources/tempo/operations/backend_search.md

```diff
@@ -12,6 +12,9 @@ some suggestions for tuning your backend to improve performance.
 General advice is to scale your compactors and queriers. Additional queriers can more effectively run jobs in parallel
 while additional compactors will more aggressively reduce the length of your blocklist and copies of data (if using RF=3).
 
+**Note:** All forms of search (TraceQL and tags based) are only supported on the `vParquet` and forward blocks. [v2 blocks]({{< relref "../configuration/parquet#disable-parquet" >}})
```

Comment:
I've added the > to make this section formatted as an admonition block. Is there additional text missing around v2 blocks? 
```suggestion
>**Note:** All forms of search (TraceQL and tags based) are only supported on the `vParquet` and forward blocks. [v2 blocks]({{< relref "../configuration/parquet#disable-parquet" >}})
```